src/install/testing/curl_grpcurl.md

```diff
@@ -51,17 +146,27 @@ grpcurl -vv -insecure \
     -d @ \
     -proto examples/protos/opentelemetry/proto/collector/logs/v1/logs_service.proto \
     -import-path examples/protos \
-    $OTLP_ENDPOINT \
+    $GRPC_OTLP_ENDPOINT \
     opentelemetry.proto.collector.logs.v1.LogsService/Export
 ```
 
-> Note: Remember to change the Service path and Service Namespace when sending other signals (traces/metrics).
+> **Note**: Remember to change the Service path and Service Namespace when sending other signals (traces/metrics).
+
+
+## Feedback 
+
+Are you experiencing issues or would like to submit a request to our team?
+* Email us at <a href="mailto:support@mydecisive.ai" target="_blank">support@mydecisive.ai</a>
+* File an issue under the <a href="https://github.com/DecisiveAI/postman/issues/new" target="_blank">MDAI Postman</a>
```

Comment:
FWIW, this should also have `rel="noopener noreferrer"`

Reply:
Oh good call. Let me do a fast-follow on all these. I bet this isn't the only place we could make this improvement.